dev-util/emscripten-fastcomp/emscripten-fastcomp-1.37.9.ebuild

```diff
@@ -40,6 +40,8 @@ src_configure() {
 	local mycmakeargs=(
 		# avoid clashes with sys-devel/llvm
 		-DCMAKE_INSTALL_PREFIX="/usr/share/${P}"
+		-DLLVM_OCAML_INSTALL_PATH="/usr/share/${P}/ocaml"
```

Comment:
Can we make this prefix compatible, putting `${ED}` at the beginning of the path?